plugins/grib_pi/src/CursorData.cpp

```diff
@@ -21,9 +15,7 @@
  *   along with this program; if not, write to the                         *
  *   Free Software Foundation, Inc.,                                       *
  *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301,  USA.         *
- ***************************************************************************
- *
- */
+ ***************************************************************************/
```

Comment:
Since you removed a Purpose: line here it might make sense to add a short file description instead. For cpp files, this is almost just a formal thing like
```
/**
* \file
* Implement CursorData.h
* / 

This applies to all cpp  files, at least those which had a Purpose line

Reply:
done as:
```c++
/**
 * \file
 * \implements \ref jsonwriter.h
 */
```